.github/ISSUE_TEMPLATE/Questions.md

```diff
@@ -4,4 +4,13 @@ about: Ask questions about the library
 
 ---
 
+Learn more about how your form project could leverage the US Forms System. To help all of us use our time most efficiently:
 
+- **What is the scope of your project?**
+- **What are you using/planning on using to build your form?**

```

Comment:
Maybe add "Include a link to your project where possible." It always helps to see if they're asking for a solution to a problem that should be solved some other way.